misago/threads/api/postingendpoint/attachments.py

```diff
@@ -31,13 +32,14 @@ class AttachmentsSerializer(serializers.Serializer):
     attachments = serializers.ListField(
         child=serializers.IntegerField(), required=False
     )
-    update_attachments = False
 
-    def validate_attachments(self, ids):
+    def __init__(self, instance=None, data=empty, **kwargs):
```

Comment:
why not `__init__(self, *args, **kwargs):`? We don't really need to know the signature here, do we?

Reply:
Just my bad habit. Changed.